lib/ronin/support/crypto/cert.rb

```diff
@@ -356,6 +357,11 @@ def self.generate(version:    2,
                             else            cert.subject
                             end
 
+          if subject_alt_names
+            extensions ||= {}
+            extensions   = extensions.merge('subjectAltName' => subject_alt_names.join(', '))
```

Comment:
Hmm, depends if we want to accept raw subject-alt-name values or IP/hostname Strings. If we accept IP/hostname strings, then we'd want to check if the string is an IP address, and map it to either `"DNS:#{value}"` or `"IP:#{value}"`.

Reply:
Alternatively, we could accept another keyword arg called `subject_alt_name` (singular) which accepts a raw `subjectAltName` extension value. So if the user wants to specify the raw value, they can use that keyword argument, otherwise `subject_alt_names` (plural) would automatically build the `subjectAltName` String.